src/oceans/models/guide.js

```diff
@@ -136,19 +136,37 @@ const guides = [
   {
     id: 'fishvtrash-pond-recall',
     textFn: () => I18n.t('fishvtrash-pond-recall'),
-    when: {appMode: AppMode.FishVTrash, currentMode: Modes.Pond},
+    when: {
+      appMode: AppMode.FishVTrash,
+      currentMode: Modes.Pond,
+      fn: state => {
```

Comment:
Why did these get added?

Reply:
I noticed that we were temporarily entering some of these states prematurely because the condition wasn't enumerated completely.  While it wasn't noticeable with a text-rendered guide, it did mean we set up to speak a phrase that would be immediately stopped again.  Tidying this up made debugging easier, too.